corrector.py

```diff
@@ -109,8 +109,8 @@ def procesar_entrega(msg):
   """
   _, addr_from = email.utils.parseaddr(msg["From"])
 
-  # Ignoramos los mails que no son para el corrector automático.
-  if GMAIL_ACCOUNT not in msg["To"]:
+  # Ignoramos los mails que vienen del sistema de entregas.
+  if "Entregas Algoritmos 2" not in msg["From"]:
```

Comment:
Esto no ignoraba ya lo que no viniera del sistema de entregas? También, por qué cambiar "Entregas Algoritmos 2" en vez de poner la cuenta como estaba antes?

Reply:
Buscando en el historial, entiendo que esto vino como consecuencia “no prevista” de https://github.com/algoritmos-rw/corrector/pull/23 y https://github.com/algoritmos-rw/algo2_sistema_entregas/pull/16. Quoting:

>Todo lo que [se busca] es poder responder a los mails del sistema de entregas y del corrector, y que lo haga a las direcciones de los alumnos, no al corrector [automático].

Pero al llegar ahora el mail por `Bcc` y no `To` (https://github.com/algoritmos-rw/algo2_sistema_entregas/commit/c0ac14696ff8477ab861f90f2092e0d600acf6cd), no es detectable de la antigua manera.

>También, por qué cambiar "Entregas Algoritmos 2" en vez de poner la cuenta como estaba antes?
Porque el corrector _también_ envía mensajes, y entraría en un loop si se los comiera.

Técnicamente, si lo queremos hacer bien, el sistema de entregas podría inyectar un header (tipo `X-Algo2-Entregas: yes` o `X-Algo2-Entregas: <token_firmado>`, y que el corrector verificase eso.

Podemos posponer este PR y hago eso cuando tenga el tiempo (total, ha estado esto sin committear tanto tiempo...); o hacer merge para que se refleje lo que hay _live_, y filear un bug. Me da igual.

Reply:
Ahhhh ok. Yo diría que no vale la pena hacerlo _tan_ robusto. Así que con este cambio estaríamos bien